### PR TITLE
fix(docs): move icons docs, point to them in README

### PR DESCRIPTION
**Pull request by StommePoes**
@aaronkaka @wyseguyonline

### CODE REVIEW

File: how-to-add-an-icon.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    To select from Font-Awesome, make sure the "select" button at the top of the page is selected (it looks like a mouse arrow in a box, has an invisible name of "select") and click on icons you'd like to add. Selected icons get an orange border and white background. Then go back to your "icomoon" group and *de-select* the whole set, so your new icons are the only selected icons on the whole page. Click the stripey bar (button with the invisible name of "Menu") to show a submenu, and choose "Copy selection to set." Now re-select  *all* the icons in your "icomoon" group (the new ones and the original ones). The sticky menu at the bottom of the screen has a link "Generate Font" which will give you both new font files and a new `selection.json` in a .zip file.
+
+3. Move the zip file to this directory (/misc/icomoon/).
+
+4. Run `icomover.sh` script with the zip file as the argument.
+
+    If you can't run the shell script, it:
+    * unpacks the zip file
+    * takes the new `selection.json` and moves it to replace the current on in `/misc/icomoon/`.
+    * renames the `icomoon.*` fonts to `pearson-fa.*` and moves those tp replace the current fonts in `/assets/fonts/`.
```
Comment (aaronkaka) [inline how-to-add-an-icon.md]: "tp" should be "to"

File: how-to-add-an-icon.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    If you run `git status` you should see the following files (if not, it means you missed one):
+
+    * `/misc/selection.json` * `/assets/fonts/pearson-fa.(svg,woff,eot,ttf versions)`
+    * `/docs/elements/icons.md`
+    * `/scss/_icons.scss`
+
+  Remove the icomoon.zip if everything was successful.
+
+6. When you PR the new icon, state in your PR message the Unicode code point for your icon (this can be found on the Font-Awesome website. Example: the bell icon is \f0f3.
```
Comment (aaronkaka) [inline how-to-add-an-icon.md]: Enclose the unicode in quotes: '\f0f3' ?

File: how-to-add-an-icon.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    To select from Font-Awesome, make sure the "select" button at the top of the page is selected (it looks like a mouse arrow in a box, has an invisible name of "select") and click on icons you'd like to add. Selected icons get an orange border and white background. Then go back to your "icomoon" group and *de-select* the whole set, so your new icons are the only selected icons on the whole page. Click the stripey bar (button with the invisible name of "Menu") to show a submenu, and choose "Copy selection to set." Now re-select  *all* the icons in your "icomoon" group (the new ones and the original ones). The sticky menu at the bottom of the screen has a link "Generate Font" which will give you both new font files and a new `selection.json` in a .zip file.
+
+3. Move the zip file to this directory (/misc/icomoon/).
+
+4. Run `icomover.sh` script with the zip file as the argument.
+
+    If you can't run the shell script, it:
+    * unpacks the zip file
+    * takes the new `selection.json` and moves it to replace the current on in `/misc/icomoon/`.
```
Comment (aaronkaka) [inline how-to-add-an-icon.md]: "on" should be "one"

File: how-to-add-an-icon.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    * renames the `icomoon.*` fonts to `pearson-fa.*` and moves those tp replace the current fonts in `/assets/fonts/`.
+
+5. Update the icons.md page with the new icon, and update `/scss/_icons.scss` with the new style.
```
Comment (aaronkaka) [inline how-to-add-an-icon.md]: "icons.md" should be "docs/elements/icons.md" to make it clear where it is located

File: how-to-add-an-icon.md (hunk 3 above)
Comment (StommePoes) [inline how-to-add-an-icon.md]: wish this said which line number

**Comment by StommePoes** [comment]
anyone else?

**Pull request merged**